docs/source/extensions/plugins.rst

```diff
@@ -4,6 +4,135 @@
 Plugins
 #######
 
-Plugins allow custom integrations to the internals of the Trainer such as a custom amp or ddp implementation.
+Plugins allow custom integrations to the internals of the Trainer such as a custom precision or
+distributed implementation.
+
+We expose Accelerators and Plugins mainly for expert users who want to extend Lightning to work with new
```

Comment:
Maybe

"Under the hood, the Lightning Trainer is using plugins in the training routine, added automatically depending on the provided Trainer arguments. For example:
.....

We expose Accelerators and Plugins mainly for expert users that want to extend Lightning for:
* New hardware (like TPU plugin)
* Distributed training algorithms (such as ...)
* Clusters (such as ....)